demo/pkg/subgraphs/demo.go

```diff
@@ -0,0 +1,33 @@
+package subgraphs
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/99designs/gqlgen/graphql"
+	"github.com/99designs/gqlgen/graphql/handler"
+	"github.com/99designs/gqlgen/graphql/handler/extension"
+	"github.com/99designs/gqlgen/graphql/handler/transport"
+	"github.com/gorilla/websocket"
+)
+
+func NewDemoServer(schema graphql.ExecutableSchema) *handler.Server {
+	srv := handler.New(schema)
+	// SSE needs to be the first transport
+	srv.AddTransport(transport.SSE{})
```

Comment:
Why? Can you explain in the comment?

Reply:
gqlgen requires it, I'd guess because other transports might incorrectly handle the request otherwise or modify the request, but I haven't dug into their code. I found it here https://gqlgen.com/recipes/subscriptions/. I've clarified the comment.